src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURE_NUMBER = 5;
+    private static final String WHITE_COLOR = String.valueOf(Colors.WHITE);
+    private static final int CIRCLE_DEFAULT_RADIUS = 10;
+
+    private static final int MAX_FIGURE_VALUES = 100;
+    private Random random = new Random();
+
+    public Figure getRandomFigure() {
+        String collor = new ColorSupplier().getRandomColor();
```

Comment:
Think about which variables should be local in the method and which should be class-level. Object `new ColorSupplier()` will be created on each call of `getRandomFigure()` method.